src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_format.py

```diff
@@ -58,16 +63,35 @@ def _format_group(item):
     """Returns an ordered dictionary of the container registry.
     :param dict item: The container registry object
     """
-    registry_info = {_registry_map[key]: str(item[key]) for key in item if key in _registry_map}
+    table_info = {_property_map[key]: str(item[key]) for key in item if key in _property_map}
 
     if 'sku' in item and 'name' in item['sku']:
-        registry_info['SKU'] = item['sku']['name']
+        table_info['SKU'] = item['sku']['name']
     if 'username' in item:
-        registry_info['USERNAME'] = item['username']
+        table_info['USERNAME'] = item['username']
     if 'passwords' in item:
         if item['passwords'] and 'value' in item['passwords'][0]:
-            registry_info['PASSWORD'] = item['passwords'][0]['value']
+            table_info['PASSWORD'] = item['passwords'][0]['value']
         if len(item['passwords']) > 1 and 'value' in item['passwords'][1]:
-            registry_info['PASSWORD2'] = item['passwords'][1]['value']
+            table_info['PASSWORD2'] = item['passwords'][1]['value']
+    # Parse webhook list-events
+    table_info['ID'] = item['id'] if 'id' in item and item['id'] and '/subscriptions/' not in item['id'] else None
+    if 'eventRequestMessage' in item and item['eventRequestMessage'] and \
+       'content' in item['eventRequestMessage'] and item['eventRequestMessage']['content']:
```

Comment:
Instead of doing multiple check you can simply try fetching `item['eventRequestMessage']['content']` and capture `KeyError`. Ask for forgiveness instead of permission is the Pythonic approach.